lesson14/main.js

```diff
@@ -71,6 +71,12 @@ const init = async () => {
   createNewList(val);
 };
 
+const getNumbers = () => {
+  const inputNumberId = document.getElementById("js-number")
+  const inputNumberValue = inputNumberId.value;
```

Comment:
NITS：

Since this variable is not going to be used for anything else in this function, 
I didn't think it was forcibly necessary to make it a variable.

Isn't it better to just return it as a .value?

Reply:
I see!
So only if we use something many times, we should make function for them, is that idea correct? :)

I've been taught before it is better to extract as much as possible that can be extracted as a function and execute it where it is used.The reason was that functions improve readability and can withstand specification changes from here on out.
I might misunderstand this meaning...

If you konw about this, would you teach me? Thanks!

Reply:
Sorry, I may not have conveyed my intentions correctly.

It's only IMO, so you don't necessarily have to do that, but

`inputNumberId.value`

This is a story it would be better to return the value as it is, instead of forcing it to be a variable. 

This value is not used frequently in the function, and it is just returned after this.
So I felt that there was no need to make it a variable.

Reply:
I just only wanted to make sure that we don't need to bother making this value variable here.

Reply:
I misunderstood, sorry about that. Now I understand what you meant.
I agree, we don't have to do if it's neccessary. I fixed that!